project/src/main/webapp/index.html

```diff
@@ -7,18 +7,18 @@
     <script src="script.js"></script>
   </head>
   <body align="center">
-        <h1> Login Page </h1>
-        <div> Username: </div>
-        <input type="text" id="username-login" name="username-login"></input>
-        <div> Password: </div>
-        <input type="text" id="password-login" name="password-login"></input><br><br>
-        <button onclick="login()">Login</button>
+        <div class="heading1"> Login Page </div><br>
```

Comment:
Why use class="heading1" and class="heading2" instead of &lt;h1&gt; and &lt;h2&gt;?